components/camel-aws/src/main/java/org/apache/camel/component/aws/sqs/SqsConfiguration.java

```diff
@@ -63,6 +63,8 @@
     private boolean extendMessageVisibility;
     @UriParam(label = "consumer", defaultValue = "1")
     private int concurrentConsumers = 1;
+    @UriParam
```

Comment:
This should be labelled as advanced. As its not a common thing to use.

Reply:
I was not aware of this possibility, that's a lot better.